pkg/controllers/cloud/networkpolicy.go

```diff
@@ -949,6 +1030,9 @@ func (a *appliedToSecurityGroup) notify(op securityGroupOperation, status error,
 		a.hasMembers = true
 	case securityGroupOperationUpdateRules:
 		// AppliedToSecurityGroup added rules, now add members.
```

Comment:
Move the comment

Reply:
I prefer have comments at the top of statements, clearly indicate the functionality of the following statements. I have updated the comment to reflect the latest changes